backend/fdo_info.mli

```diff
@@ -33,12 +33,12 @@
 type info = private
   {
     dbg: Debuginfo.t;
-    discriminator: int;
+    discriminator: InstructionId.t;
```

Comment:
The `discriminator` should remain as an int (fdo puts block label id there).

Reply:
So the discriminator is either a label or an instruction id?

Reply:
yes